src/Scripting/GraphTypes.hpp

```
@@ -0,0 +1,217 @@
+		if (type == TID_OBJ)		return AsTL::TID_IDX;
+		else if (type == TID_SPC)	return AsTL::TID_IDX;
+		else if (type == TID_BODY)	return AsTL::TID_IDX;
+		else if (type == TID_SPK)	return AsTL::TID_STR;
+		else if (type == TID_EXEC)	return AsTL::TID_BOOL;
+
+		else return type;
+	}
+
+
+	/* Converts a serialized string to a pin type. */
+	inline static std::type_index StringToPinType(const std::string &type) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)